Sentinel/Classes/DatabaseImportExport/Internal/ZipHandler.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ZipHandler.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 13.03.2025..
+//
+
+import Foundation
+
+enum CreateZipError: Error {
+    case urlNotADirectory(URL)
+    case failedToCreateZIP(Error)
+}
+
+enum ZipHandler {
```

Comment:
Maybe go with `Archive` instead of `Zip`? I'm fine with both, but a bit creative today 😄